apps/condo/domains/resident/schema/SuggestServiceProviderService.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Generated by `createservice organization.SuggestServiceProviderByTinService --type queries`
+ */
+
+const { pick, uniqBy } = require('lodash')
+
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { MeterResourceOwner } = require('@condo/domains/meter/utils/serverSchema')
+const { SERVICE_PROVIDER_TYPE } = require('@condo/domains/organization/constants/common')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const access = require('@condo/domains/resident/access/SuggestServiceProviderService')
+const { MAX_RESIDENT_SUGGEST_SERVICE_PROVIDER_BY_WINDOW_SEC, RESIDENT_SUGGEST_SERVICE_PROVIDER_WINDOW_SEC } = require('@condo/domains/resident/constants')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+const redisGuard = new RedisGuard()
+
+const checkLimits = async (uniqueField) => {
+    await redisGuard.checkCustomLimitCounters(
+        `suggest-service-provider-by-tin-${uniqueField}`,
```

Comment:
Also strange naming convention, we use underscore for multi-words and `:` for parts delimiter

Reply:
Also note that:
1. redis guard by default has its own prefix guard_counter:
2. unique field must be following model name and be as close to beginning as possible

Reply:
Check other usages of checkCustomLimitCounters in domains folder. There they use `-` as delimiter and puts unique key at the end. Maybe pass this now, and refactor all at once later?